<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-F2PLA7Q6" failures="3" tests="3" name="BankingApp Test(failed)" time="49.967" errors="0" timestamp="2022-02-25T11:40:26 PKT">
  <testcase classname="testCases.TC_LoginDDT_002" name="loginDDT" time="7.871">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="testCases.TC_LoginDDT_002" name="loginDDT" time="8.472">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="testCases.TC_LoginDDT_002" name="loginDDT" time="13.675">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- BankingApp Test(failed) -->
